HACKING.md

```diff
@@ -51,6 +51,8 @@ required.
 
 Scylla is built with [Ninja](https://ninja-build.org/), a low-level rule-based system. A Python script, `configure.py`, generates a Ninja file (`build.ninja`) based on configuration options.
 
+**Note**: On ubuntu it is possible that the installation directory will be under `/usr/bin/ninja` and CMake search this under `/usr/bin/ninja-build` so a possible sulotion can be using symlink. `ln -s /usr/bin/ninja /usr/bin/ninja-build`
+
```

Comment:
I don't see that, installing ninja-build gives me /usr/bin/ninja.

Reply:
Have you used apt-get install ninja-build on ubuntu ? I tried it on ubuntu docker.

Reply:
yes

Reply:
OK then, I'll close this PR. I guess I do something weird.